packages/itwin/one-click-lca-widget/src/api/reporting/openapi/reporting.json

```diff
@@ -0,0 +1,301 @@
+{
+  "openapi": "3.0.1",
+  "info": {
+    "title": "Reporting",
+    "description": "Insights is a resource for aggregating iTwin data to build custom dashboards and integrate iTwin data with your business applications.\r\n\r\n- Create digitial twin based reports which can be consumed through business intelligence applications such as Power BI.\r\n- Leverage the OData feed Data Access to generate reports by combining data from multiple data streams in the digital twin.",
+    "version": "v1"
+  },
+  "servers": [
+    {
+      "url": "https://sbx-api.bentley.com/insights/reporting"
```

Comment:
This is a duplicate, and different, than the one that was already made over in the [grouping-mapping widget](https://github.com/iTwin/viewer-components-react/blob/master/packages/itwin/grouping-mapping-widget/src/api/openapi/reporting.json). Has it been discussed creating a client package that can be shared between the components, and anyone else who wants to use the API?

Reply:
We are working on creating a client package first. Will come back to resolve this later